GNUmakefile

```diff
@@ -41,13 +41,9 @@ cov:
 	gocov test ${GOFILES} | gocov-html > /tmp/coverage.html
 	open /tmp/coverage.html
 
-test:
-	@./scripts/verify_no_uuid.sh
-	@env \
-		GOTAGS="${GOTAGS}" \
-		GOFILES="${GOFILES}" \
-		TESTARGS="${TESTARGS}" \
-		sh -c "'$(CURDIR)/scripts/test.sh'"
+test: dev
+	go test -tags "$(GOTAGS)" -i -run '^$$' ./...
+	go test -tags "$(GOTAGS)" -v $$(go list ./... | grep -v vendor) | tee test.log 2>&1
```

Comment:
There's a `GOFILES` thing we could use here as well as `TEST` for the above (to get `./...` but let it be hooked into).